frontend/components/shared/buttons/Button.less

```diff
@@ -0,0 +1,24 @@
+@import '../../../styles/Variables.less';
+@import '../../../styles/Extends.less';
+
+.custom-button:extend(.btn) {
+  width: max-content;
+  font-weight: 700;
+  text-decoration: none;
+}
+
+.custom-button.primary-button {
+  background-color: @accents-and-btn;
+  color: @white;
+}
+
+.back-to-btn {
+  padding: 10px 20px;
+  width: max-content;
+}
+
+.back-to-btn > a {
```

Comment:
a {
...styles
}

should also work inside `.back-to-btn`

Reply:
Yes, but I want to add this style only for a inside .back-to-btn, not for all